package-lock.json

```diff
@@ -9447,13 +9447,13 @@
       }
     },
     "node_modules/postcss": {
-      "version": "8.3.11",
-      "resolved": "https://registry.npmjs.org/postcss/-/postcss-8.3.11.tgz",
-      "integrity": "sha512-hCmlUAIlUiav8Xdqw3Io4LcpA1DOt7h3LSTAC4G6JGHFFaWzI6qvFt9oilvl8BmkbBRX1IhM90ZAmpk68zccQA==",
+      "version": "8.4.4",
```

Comment:
`postcss-html@1.3.0` requires `postcss@>=8.4.0`. This update to PostCSS 8.4 is in progress with PR #5725.